src/components/InCallWaitingRoom.jsx

```diff
@@ -0,0 +1,137 @@
+import { useRef } from "react";
+import styled from "styled-components";
+import HeaderText from "../components/text/HeaderText";
+import checkmark from "../components/images/checkmark.svg";
+import {
+  Icon,
+  HintList,
+  HintListItem,
+  SubContainer,
+  InstructionText,
+  FormHeader,
+  HintListItemText,
+} from "../components/List";
+import theme from "../theme";
+
+const InCallWaitingRoom = ({ startTime, error, joinCall }) => {
+  const inputRef = useRef(null);
+
+  const handleSubmitNameForm = (e) => {
+    e.preventDefault();
+    if (!inputRef?.current) return;
+
+    const username = inputRef.current.value?.trim();
+    joinCall(username);
+  };
+
+  return (
+    <Container>
+      <SubContainer>
+        <HeaderText>Welcome!</HeaderText>
+        <InstructionText>
+          Here are some things to know before we get started:
+        </InstructionText>
+        <HintList>
+          {startTime && (
+            <HintListItem>
+              <Icon src={checkmark} alt="checkmark" />
```

Comment:
I think these alt-texts is not valuable for people using screen readers if HintList is a `<ul>`.